File: dynamic_form_gen/src/main/java/com/ihsinformatics/dynamicformsgenerator/data/pojos/Workflow.java
package com.ihsinformatics.dynamicformsgenerator.data.pojos;

import com.google.gson.annotations.SerializedName;

public class Workflow {

    @SerializedName("uuid")
    private String uuid;

    //    @PrimaryKey
    @SerializedName("workflowId")
    private long workflowId;

    @SerializedName("name")
    private String name;

    public String getUuid() {
        return uuid;
    }

    public void setUuid(String uuid) {
        this.uuid = uuid;
    }

    public long getWorkflowId() {
        return workflowId;
    }

    public void setWorkflowId(long workflowId) {
        this.workflowId = workflowId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/ihsinformatics/com/hydra_mobile/data/local/entities/workflow/WorkFlow.kt
package ihsinformatics.com.hydra_mobile.data.local.entities.workflow

import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.SerializedName

@Entity
class WorkFlow(

    @SerializedName("uuid")
    val uuid: String,
    @PrimaryKey
    @SerializedName("workflowId")
    val workflowId: Int,
    @SerializedName("name")
    val name: String
)